client/landing/stepper/declarative-flow/free.ts

```diff
@@ -30,7 +35,6 @@ import {
 	Flow,
 	ProvidedDependencies,
 } from './internals/types';
-import type { OnboardSelect, UserSelect } from '@automattic/data-stores';
```

Comment:
Was this something a linter/formatter changed? Not blocking at all, as I am used to having the types imported separately.

Reply:
Actually, it was the VS code that applied these changes when I imported the function.